README.md

```diff
@@ -12,97 +12,26 @@ OSTIS Web platform contains:
 
 To learn more about the platform, check out our [documentation](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
 
-## Installation
+## Installation and usage
 
-- Quick start using Docker Compose
```

Comment:
Why do you remove docker installation. It is just another way to install project. Revert it please